lib/cocoapods/installer/user_project_integrator/target_integrator/xcconfig_integrator.rb

```diff
@@ -29,37 +28,6 @@ def self.integrate(pod_bundle, targets)
           # @!group Integration steps
           #-------------------------------------------------------------------#
 
-          # Removes the xcconfig used up to CocoaPods 0.33 from the project and
-          # deletes the file if it exists.
-          #
-          # @param  [Target::AggregateTarget] pod_bundle
-          #         The Pods bundle.
-          #
-          # @param  [Array<XcodeProj::PBXNativeTarget>] targets
-          #         The native targets.
-          #
-          # @todo   This can be removed for CocoaPods 1.0
```

Comment:
oh yeah, given it's a major version change its also safe (since major version changes blow away the sandbox)